source/_components/sensor.llap.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "LLAP Sensor"
+description: "Instructions how to integrate Ciseco / Wireless Things sensors into Home Assistant."
+date: 2017-02-15 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.39
+ha_iot_class: "Local Push"
+logo: wirelessthings.png
+---
+
+The `llap` platform supports wireless sensors which were produced by now defunct Ciseco / Wireless
```

Comment:
I usually put line breaks so that source file is also readable in plain text editors.